requirements.txt

```diff
@@ -15,3 +15,5 @@ pycryptodomex
 
 # TODO: Building with a custom openssl 1.1.1 on centos 6 is failing to compile M2Crypto, this should be fixed in the future as M2Crypto is prefered
 #M2Crypto>=0.35.2
+
+pop-build
```

Comment:
This requirements.txt file is used to build Salt. pop-build is not used in Salt, so it should not go in this file